src/index.html

```diff
@@ -4,13 +4,42 @@
     <meta charset="UTF-8">
     <meta
       name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
+      content="width=device-width, user-scalable=no, initial-scale=1.0,
+      maximum-scale=1.0, minimum-scale=1.0"
     >
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Search bar airbnb</title>
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <div class="bigSearch">
+      <form
+        action="/"
```

Comment:
When i press enter I get the 404 page, fix action
![image](https://user-images.githubusercontent.com/37161315/136955196-6251f817-ba6f-4538-b0d5-8e34d86fdaa7.png)